console/templates/submit.html

```diff
@@ -8,18 +8,18 @@
 
   {% include 'partials/navigation_tabs.html' %}
 
-  <div class="alert hidden alert-danger" role="alert" style="overflow: auto">
-    <div class="alert hidden alert-danger panel__body" role="alert">
-      <div class="alert hidden alert-danger" role="alert">
-        {# <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">×</span></button> #}
-      </div>
-    </div>
-  </div>
+    <div class="container">
 
-  {% include 'partials/submitter.html' %}
+    {% include 'partials/submitter.html' %}
 
-  {% include 'partials/receiver.html' %}
+      <div class="alert hidden alert-danger" role="alert" style="overflow: auto">
+        <div class="alert hidden alert-danger panel__body" role="alert">
+          <div class="alert hidden alert-danger" role="alert">
+            {# <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">×</span></button> #}
```

Comment:
Is this a comment?  If so, can you remove it.  If not, can you let me know what it's doing because I'm not really sure